### ISSUE TITLE
Website/Transferring Lobby Advertisements

### ISSUE THREAD

**Issue by colton-boi** [Minehut/Meta#935]
### Feature request

**Checklist**
- [x I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
 Website/Ingame

**Describe the feature**
Cheap advertisement for small servers similar to Roblox's system. You can purchase an ad spot on the website OR in the transferring lobby, once your advertisement gets approved, for credits/money

**How the feature is useful**
This would benefit servers that are trying to grow and could help slowly phase away from the horrible /ad spam in the hubs. It would also benefit MH making more money

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by hapily04** [comment]
I don't think this would work well. This would easily become backlogged, completely destroying the purpose of getting players quickly.

**Comment by Tringlle** [comment]
I mean, it's possible. Purchasing ads on the website / in-game? Sponsors do it, so I don't see why we can't? You'd just have to pay real money, not credits.
